partner_purchase_multicompany_default/readme/ROADMAP.rst

```diff
@@ -0,0 +1,9 @@
+Since this module propagates product accounts based on the account code, it
```

Comment:
to be droppped?

Reply:
Still relevant, as an account code is not the same accross all countries.